model/ftn/w3iogomd.ftn

```diff
@@ -1492,7 +1496,6 @@
 ! Old definitions:  MSCX(JSEA)  = ABX2(JSEA) * FACTOR * FACTOR2
 !                   MSCY(JSEA)  = ABY2(JSEA) * FACTOR * FACTOR2
             MSCD(JSEA)=0.5*ATAN2(2*MB,MA-MC)
-
```

Comment:
@erinethomas, this looks good. The only comment I have is to get rid of the whitespace changes.

Reply:
done.